Cargo.toml

```diff
@@ -25,6 +25,13 @@ mitosis = {version = "0.1.1", optional = true}
 log4rs = "1.3.0"
 bdk = "0.29.0"
 
+bdk_wallet= {git = "https://github.com/bitcoindevkit/bdk", subdir="crates/wallet",  rev = "3b040a7ee69c7fefac6c3ab57c17b220e0beb9fb"}
+bdk_persist= {git = "https://github.com/bitcoindevkit/bdk", subdir="crates/bdk_persist",  rev = "3b040a7ee69c7fefac6c3ab57c17b220e0beb9fb"}
+bdk_chain= {git = "https://github.com/bitcoindevkit/bdk", subdir="crates/bdk_chain",  rev = "3b040a7ee69c7fefac6c3ab57c17b220e0beb9fb"}
+
+strum_macros = "0.26.4"
+strum = "0.26.2"
+miniscript = "12.0.0"
```

Comment:
There is a miniscript export from bdk_chain via the `miniscript` feature. We can use that. That will later on simplify our life for dependency version conflicts. 

both bitcoin and miniscript can be used from bdk_chain.